### PR TITLE
refactor: Replace some TODO items with issue ids

### PR DESCRIPTION
**Pull request by zharinov**
## Changes:

Nothing should be changed

## Context:

This PR covers TODOs that aren't related to type issues.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/manager/travis/update.spec.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@ const content = loadFixture(__filename, 'travis.yml');
 describe(getName(__filename), () => {
   describe('updateDependency', () => {
     it('updates values', () => {
-      // TODO: should be `Upgrade`
```
Comment (zharinov) [inline lib/manager/travis/update.spec.ts:9]: https://github.com/renovatebot/renovate/issues/9647

File: lib/manager/travis/update.ts [hunk 2]
```diff
@@ -20,7 +20,6 @@ export function updateDependency({
       quote = '';
     }
     let newString = `node_js:\n`;
-    // TODO: `newValue` is a string!
```
Comment (zharinov) [inline lib/manager/travis/update.ts:23]: https://github.com/renovatebot/renovate/issues/9647

**Comment by zharinov** [comment]
Just like "bad candies" accompany our New Year celebration being left intact during Christmas, the remaining TODO items look hard for me to formalize. Therefore, this PR is ready for review.

**Comment by rarkins** [comment]
What do you think about meaning the description in place and a short issue number after it (eg `(#1234)`)

**Review by rarkins — APPROVED** [review]

**Pull request merged**